js/privatebin.js

```diff
@@ -4818,16 +4851,44 @@ jQuery.PrivateBin = (function($, RawDeflate) {
          * @param {int} status
          * @param {object} data
          */
-        function showCreatedPaste(status, data) {
+        async function showCreatedPaste(status, data) {
             Alert.hideLoading();
             Alert.hideMessages();
 
             // show notification
-            const baseUri   = Helper.baseUri() + '?',
-                  url       = baseUri + data.id + '#' + CryptTool.base58encode(data.encryptionKey),
-                  deleteUrl = baseUri + 'pasteid=' + data.id + '&deletetoken=' + data.deletetoken;
+            const baseUri   = Helper.baseUri(),
+            deleteUrl = baseUri + '?pasteid=' + data.id + '&deletetoken=' + data.deletetoken;
+            let url;
+
+            const pw = TopNav.getPassword()
+
+            // only execute when it is a single time view
+            if(pw && pw.length && TopNav.getBurnAfterReading()) {
+
+                const sm =  CryptTool.getSymmetricKey();
+
+                let openUri = '?' + data.id + '#' + CryptTool.base58encode(data.encryptionKey);
+                let cipherResult = await CryptTool.cipher(sm, pw, openUri, []);
+
+                let dt = {}
+                dt['v'] = 2;
+                dt['ct'] = cipherResult[0];
+                dt['adata'] = cipherResult[1];
+                dt['k'] = sm;
+
+                const encUrl = CryptTool.base58encode(JSON.stringify(dt))
+
+                url = baseUri +  '#protected/' + encUrl;
+            } else {
+                url       = baseUri + '?' + data.id + '#' + CryptTool.base58encode(data.encryptionKey);
```

Comment:
```suggestion
                url = baseUri + '?' + data.id + '#' + CryptTool.base58encode(data.encryptionKey);
```

Reply:
We do have such rules and they are enforced using StyleCI / github action. This particular case was ignored by it and this PR passed the test. It's nice that it got spotted in this review.

Reply:
Good to hear, I think the autoformatter indeed did run. B/c we wouldn't have changed this line https://github.com/PrivateBin/PrivateBin/pull/1231/files#diff-4c30f267da1a33c8d947890833316f48f2c58b89bc0ae567a469056ce98b96e5R231 . That was most likely the auto formatter following the rules.